### ISSUE TITLE
Screen reader doesn't work with output content

### ISSUE THREAD

**Issue by chrispaynter** [CharlesMangwa/react-native-simple-markdown#73]
Hi, I've just tested an app I'm building with a screen reader, and it seems that this plugin is incompatible.

For example, the user has to navigate through the text word by word, because the text seems to be broken into seperate text components. Also, even if they were to navigate, the text doesn't read out in consecutive order, the reading basically becomes a garbled nonsensical paragraph.

Does anyone know how I can fix this? Hoping that swapping out the plugin is the solution...

**Comment by CharlesMangwa** [comment]
Maybe this lib insn't compatible with what you're working on indeed. But feel free to join us [here](#75) and share what you'd like to have with RNSM!
